### ISSUE TITLE
Kari Comments

### ISSUE THREAD

**Issue by karinorman** [FrancisBanville/ms_interpreting_probabilities#27]
Wow @FrancisBanville! I am so impressed by this! The writing is clear and concise and the whole thing is really well structured. I've given some comments, hopefully not too nitpicky, as a reader who is interested in network concepts but not necessarily immersed in the literature. I couldn't get to the whole thing this morning but I'll come back to it again tomorrow!

line 4-5: "of which the vast majority are rare" -> "the vast majority of which are rare" (take or leave this one but the second sounds a lot better to my ear :) )

line 14-16: These feel more like the factors that determine whether they encounter locally (the second condition), rather than it's own independent condition

line 25-26: This second half of the sentence is hard to follow. I think the scale piece needs its own sentence. Also you probably did it intentionally, but "degree of belief" as the definition of probability is distinctly bayesian some people might be a little confused by that definition being used right out of the gate if they haven't been exposed to it before.

line 27: You might want to be a little more precise here with "ecological data" because everything you've described so far hasn't been uncertainity/variablity in data, but rather the underlying ecological processes

line 32-34: I understand why it's feels natural to put this here, but again I think the concept of scale might need it's own paragraph. It feels a little out of place here

line 51: I think you need to define topological null models

line 122: Does "these" refer to interaction strengths? It's not obvious whether the antecedent is binary networks or interaction strengths

line 127: Is "quantitative interaction" an accepted term in the field? If not, it feels pretty ambiguous. Both interaction strengths and probabilities are quantitative, but I think you're just referencing interaction strengths here?
- reading on in the paragraph it seems clear you're just talking about networks with interaction strengths, in which case I think you need a sentence explicitly defining the relationship between interaction strengths, probabilities, and quantitative networks

line 184: Do they impact our _estimation_ or do they just impact the probability? I think the latter?
- ah, I see what you mean now after reading on, that the context of the data may impact the estimated probability. I still think it's not precisely true that those factors impact estimation :) 

line 197: I really like this concept as almost a partitioning of the probability

**Comment by FrancisBanville** [comment]
Thanks @karinorman! I agree that the points you highlighted need a bit more precision. Looking forward to seeing your other suggestions tomorrow!

**Comment by FrancisBanville** [comment]
Did you have any other comments finally @karinorman?

**Comment by FrancisBanville** [comment]
Ok thanks! Hope the family is feeling better
